chapters/envs.rst

```diff
@@ -14,157 +16,168 @@ Here is the list of environment variables that ``jina`` respects during runtime.
 
     :default: unset
 
-.. confval:: JINA_VCS_VERSION
+.. confval:: JINA_CONTRIB_MODULE_IS_LOADING
 
-    Git version of ``jina``. This is used when ``--check-version`` is turned on. For official docker image of ``jina``, ``JINA_VCS_VERSION`` is automatically set to the git version during the building procedure.
+    Set to True when loading the third party components.
 
-    :default: the git head version for ``jina`` image. If you are using ``jina`` locally outside docker container then this is unset.
+    :default: unset
 
 .. confval:: JINA_CONTROL_PORT
 
     Control port of all pods.
 
     :default: unset. A random port will be used for each :func:`BasePod`.
 
+.. confval:: JINA_DEFAULT_HOST
 
-.. confval:: JINA_CONTRIB_MODULE
+    The default host address of Jina.
 
-    Paths of the third party components.
+    :default: `0.0.0.0`
+
+.. confval:: JINA_DISABLE_UVLOOP
+
+    ``uvloop`` is ultra fast implementation of the asyncio event loop on top of libuv. Since Jina 0.3.6, Jina relies on ``uvloop`` to manage the sockets and streams.
+
+    :default: unset
+
+.. confval:: JINA_EXECUTOR_WORKDIR
+
+    The default executor working directory, where dumping and IO output happens.
+
+    :default: unset
+
+.. confval:: JINA_FULL_CLI
+
+    When set, show all Jina subcommands.
 
     :default: unset
 
 .. confval:: JINA_IPC_SOCK_TMP
 
     Temp directory when using IPC sockets for the control port, not used on Windows system or when the control port is over TCP sockets.
 
-.. confval:: JINA_LOG_FILE
+.. confval:: JINA_LOG_CONFIG
 
-    Control where the logs output to. Possible values: ``TXT`` for plain text format, which is the uncolored version of the screen output. ```JSON`` for structured log output.
+    The configuration file for the logger. If unset then will load from logging.default.yml.
 
-    :default: unset, aka stdout only.
+    :default: unset
 
-.. confval:: JINA_SOCKET_HWM
+.. confval:: JINA_LOG_ID
 
-    High-watermarks of ZMQ send & receive sockets. Reference: http://api.zeromq.org/master:zmq-setsockopt
+    The identifier of a logger so that it can be used as group identifier by ``fluentd``. It is set when Pea starts a new process
+    to allow grouping by pod identifier for pea and executor logger.
 
-    :default: 4
+    :default: unset
 
-.. confval:: JINA_ARRAY_QUANT
+.. confval:: JINA_LOG_LEVEL
 
-    Quantization scheme when storing ndarray into protobuf message, useful for reducing the network latency and saving bandwidth. Possible values: ``fp16`` (almost lossless), ``uint8``.
+    The log verbosity of the Jina logger. Possible values are ``DEBUG``, ``INFO``, ``SUCCESS``, ``WARNING``, ``ERROR``, ``CRITICAL``.
 
-    :default: unset
+    :default: ``INFO``
 
 .. confval:: JINA_LOG_NO_COLOR
 
     Show colored logs in stdout, set to any non-empty value to disable the color log, e.g. if you want to pipe the log into other apps.
 
     :default: unset
 
-.. confval:: JINA_EXECUTOR_WORKDIR
+.. confval:: JINA_LOG_WORKSPACE
```

Comment:
where is this used?

Reply:
JINA_LOG_WORKSPACE  is used in optimizer/discovery.py and mostly used in tests, examples.